src/app/Game.js

```diff
@@ -0,0 +1,98 @@
+            {"name":"Spaghetti carbonara","imgSrc":"/static/assets/img/carbonara.jpg","apiTitle":"spaghetti+carbonara"},
+            {"name":"Cheesecake","imgSrc":"/static/assets/img/cheesecake.jpg","apiTitle":"cheesecake"},
+            {"name":"Chicken pad thai","imgSrc":"/static/assets/img/chicken-pad-thai.jpg","apiTitle":"chicken+pad+thai"},
+            {"name":"Chocolate brownie","imgSrc":"/static/assets/img/chocolate-brownie.jpg","apiTitle":"chocolate+brownie"},
+            {"name":"Hamburger","imgSrc":"/static/assets/img/hamburger.jpg","apiTitle":"Hamburger"},
+            {"name":"Spaghetti aglio et olio","imgSrc":"/static/assets/img/spaghetti-aglio-olio.jpg","apiTitle":"spaghetti+aglio+olio"},
+            {"name":"Spaghetti bolognese","imgSrc":"/static/assets/img/spaghetti-bolognese.jpg","apiTitle":"spaghetti+bolognese"},
+            {"name":"Strawberry shake","imgSrc":"/static/assets/img/strawberry-shake.jpg","apiTitle":"strawberry+shake"},
+            {"name":"Vanilla ice cream","imgSrc":"/static/assets/img/vanilla-ice-cream.jpg","apiTitle":"vanilla+ice+cream"},
+        ]
+        this.startBtn = document.querySelector(".start")
+        this.instructionSection = document.querySelector(".instruction")
+        this.gameSection = document.querySelector(".game")
+        this.imgInput = document.querySelector(".dish-img")
+        this.dishNameInput = document.querySelector(".dish-name")
+        this.checkBtn = document.querySelector(".submit")
+        this.answerInput = document.querySelector(".answer")
+        this.correctAnswer = document.querySelector(".correct-answer")
+        this.gameOverBlock = document.querySelector(".game-over")
+        this.finalScoreInput = document.querySelector(".game-over .score span")
+
+        this.stats = new Stats(0)
+        
+        document.addEventListener('DOMContentLoaded', () => {
+            this.startBtn.addEventListener("click", this.startRound.bind(this), {once: true})
+            this.checkBtn.addEventListener("click", (e) => e.preventDefault())
+        })
+    }
+
+    showQuestionSection() {
+        this.gameSection.classList.add("active")
+        this.instructionSection.classList.remove("active")
+    }
+
+    async startRound() {
+        const question = new Question(this.questions)
+        const timer = new Timer(15)
+        if (!this.gameSection.classList.contains("active")) this.showQuestionSection()
+        const currentQuestion = await question.getQuestion()
+        this.reset()
+        this.imgInput.src = await currentQuestion.imgSrc
+        this.imgInput.alt = await currentQuestion.name
+        this.dishNameInput.textContent = await currentQuestion.name
+        timer.startTimer(await question, this)
+        
+        this.sumUpRoundFunction = this.sumUpRound.bind(this, question, timer)
+        this.checkBtn.addEventListener("click", this.sumUpRoundFunction, {once: true})  
+    }
+
+    sumUpRound(question, timer, e) {
```

Comment:
Radek said that we should avoid one letter variables, even when they are obvious